auth/auth-router.js

```diff
@@ -1,11 +1,55 @@
-const router = require('express').Router();
+const router = require("express").Router();
+const Users = require("./authHelper");
+const secrets = require("../config/secrets");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
 
-router.post('/register', (req, res) => {
-  // implement registration
+router.post("/register", async (req, res) => {
+  let user = req.body;
+  console.log(user);
+  const hash = bcrypt.hashSync(user.password, 10);
+  user.password = hash;
+  const addedUser = await Users.add(user);
```

Comment:
You want to check if the user has sent all of the information before you make a call to the database or hash the password. If they don't send a password, your server might shut down since you would be passing the hash function undefined. 

Also, You want to keep any functions with await inside of your try catch. If the promise fails, it would fall to the catch statement. The way you have it set up might still go to the catch statement. But the logic makes more sense to have the promises inside of the try.